toolkit/scripts/toolchain/container/toolchain_build_in_chroot.sh

```diff
@@ -67,14 +67,14 @@ set -e
 #
 cd /sources
 
-echo Linux-5.15.45.1 API Headers
-tar xf kernel-5.15.45.1.tar.gz
-pushd CBL-Mariner-Linux-Kernel-rolling-lts-mariner-2-5.15.45.1
+echo Linux-5.15.48.1 API Headers
```

Comment:
In my opinion we should reduce typing as much as possible :)
How about:
```suggestion
KERNEL_VERSION="5.15.48.1"
echo Linux-${KERNEL_VERSION} API Headers
```